code/world.dm

```diff
@@ -22,3 +22,8 @@
 #ifdef FIND_REF_NO_CHECK_TICK
 	loop_checks = FALSE
 #endif
+
+#if DM_BUILD >= 1506
+// We don't actually care about storing the output here, this is just an easy way to ensure the profile runs first.
+GLOBAL_REAL_VAR(world_init_profiler) = world.Profile(PROFILE_START)
```

Comment:
This is not a static value, and as such won't compile.

Reply:
not the issue. try again

Reply:
Looks like `world.dm` is read before `\_DEFINES\global.dm`, so the macro is not yet defined when this code is executed.

Reply:
I think I'd put this in `__DEFINES/_profiler_setup.dm`

Reply:
I was thinking about just doing `var world_init_profile = ...` =P

Reply:
Doesn't matter too much really. Someone is aiming to improve file layout/organization soon, so we'll see how that goes.